dev/tools/update_icons.dart

```diff
@@ -50,15 +57,6 @@ const Map<String, String> _identifierRewrites = <String, String>{
   '8k': 'eight_k',
   '9k': 'nine_k',
   '10k': 'ten_k',
-  '1k_plus': 'one_k_plus',
-  '2k_plus': 'two_k_plus',
-  '3k_plus': 'three_k_plus',
-  '4k_plus': 'four_k_plus',
-  '5k_plus': 'five_k_plus',
-  '6k_plus': 'six_k_plus',
-  '7k_plus': 'seven_k_plus',
-  '8k_plus': 'eight_k_plus',
-  '9k_plus': 'nine_k_plus',
```

Comment:
what happened to these? Is removing them going to break any code?

Reply:
These are extraneous, already covered by rewrites for 1k to 10k

Reply:
oh, nice, I hadn't noticed this map is being used to do prefix matching rather than just complete matching. Can you put a comment at the top of this map that mentions that? That would be helpful for future edits.

Reply:
Sure, done